lib/administrate/version.rb

```diff
@@ -1,3 +1,3 @@
 module Administrate
-  VERSION = "0.2.0".freeze
+  VERSION = "0.2.1".freeze
```

Comment:
actually it is a requirement to get this to work, otherwise you get compatibility issues since version 0.2.0 has dependencies as defined previously to this work. Upgrading rails with cross deps is pretty annoying, and bundler is pretty dumb to the change, hence the version bump. This PR is the review; i'm sure things will get edited out for merge.